leasing/migrations/0060_fix_paikkatietovipunen_vuokraalueet_remove_intended_use_requirement.sql

```diff
@@ -1,5 +1,5 @@
 -- View: public.paikkatietovipunen_vuokraalueet
--- DROP VIEW public.paikkatietovipunen_vuokraalueet;
+DROP VIEW public.paikkatietovipunen_vuokraalueet;
```

Comment:
One last thing regarding changing past migrations. You won't be able to test that very easily.
Would it be better to add this DROP VIEW in the migration you created before running the file instead of modifying the old file?
It is a small thing but might make things more reliable